libreport.spec.in

```diff
@@ -54,6 +57,7 @@ BuildRequires: xz
 BuildRequires: lz4
 Requires: libreport-filesystem = %{version}-%{release}
 Requires: satyr >= 0.18
+Requires: glib2 >= %{glib_ver}
```

Comment:
Are you sure this explicit library requires is needed? rpmbuild should be able to add the dependency automatically:
https://fedoraproject.org/wiki/Packaging:Guidelines#Explicit_Requires